CONTRIBUTING.md

```diff
@@ -584,3 +533,40 @@ $$\alpha = \frac{\beta}{\gamma}$$
 
 Example of auto-built documentation for `webviz-config` can be seen
 [here on github](https://equinor.github.io/webviz-config/).
+
+## Make your plugin project available
+
+It is strongly recommended to store your plugin project code base in a `git` solution,
+e.g. [in a new GitHub repository](https://github.com/new). If possible, it is also
+_highly_ recommended to let it be an open source repository. This has several advantages:
+ - Others can reuse your work - and help each other achieve better code quality, add more features and share maintenance
+ - You will from the beginning make sure you keep a good separation between visualization code and raw data to be visualized.
```

Comment:
replace `and raw data to be visualized` with e.g. `and data loading/processing code`